docs/windows/BUILD.md

```diff
@@ -79,20 +73,22 @@ git checkout infra/refactor
 
 ### Option 1: Build Everything (Recommended)
 
+Use the provided build script (preferred over make targets):
+
 ```bash
-# On Linux/macOS
-make -f Makefile.windows windows-all
+# On Windows (Git Bash or cmd.exe)
+./build-windows.bat
 
-# On Windows
-mingw32-make -f Makefile.windows windows-all
+# On Linux/macOS (cross-compile)
+./build-windows.sh
```

Comment:
This is only valid once https://github.com/Cypherpunk-Labs/PinShare/pull/3#discussion_r2590336902 is addressed.

Reply:
Good catch. Added a note that the cross-compilation script (build-windows.sh) requires additional testing before it can be documented as a supported option.